config.go

```diff
@@ -131,18 +131,20 @@ type ResourcesRequestsOptions struct {
 // Config represents the serialized state of a Function's metadata.
 // See the Function struct for attribute documentation.
 type Config struct {
-	Name        string            `yaml:"name" jsonschema:"pattern=^[a-z0-9]([-a-z0-9]*[a-z0-9])?$"`
-	Namespace   string            `yaml:"namespace"`
-	Runtime     string            `yaml:"runtime"`
-	Image       string            `yaml:"image"`
-	ImageDigest string            `yaml:"imageDigest"`
-	Builder     string            `yaml:"builder"`
-	BuilderMap  map[string]string `yaml:"builderMap"`
-	Volumes     Volumes           `yaml:"volumes"`
-	Envs        Envs              `yaml:"envs"`
-	Annotations map[string]string `yaml:"annotations"`
-	Options     Options           `yaml:"options"`
-	Labels      Labels            `yaml:"labels"`
+	Name            string            `yaml:"name" jsonschema:"pattern=^[a-z0-9]([-a-z0-9]*[a-z0-9])?$"`
+	Namespace       string            `yaml:"namespace"`
+	Runtime         string            `yaml:"runtime"`
+	Image           string            `yaml:"image"`
+	ImageDigest     string            `yaml:"imageDigest"`
+	Builder         string            `yaml:"builder"`
+	BuilderMap      map[string]string `yaml:"builderMap"`
```

Comment:
Having both a Builder and Builders is perhaps marginally better than appending "Map" to the plural:  Not only is it a bit closer to natural language, it allows us to match `Config` closely with the `Manifest` (one step closer to being an embedded struct).

While there is of course the possibility of confusion, I consider plurality to be an important in establishing member uniqueness.

Throwing caution and potential confusion to the wind, the subtle plurality differentiator is already being embraced with the members `repository` vs `repositories` on the Client.  Perhaps its worth considering here as well.

Reply:
> While there is of course the possibility of confusion, I consider plurality to be sufficient for establishing uniqueness, despite the diminutive size of the poor little "s".

:rofl: I agree and will change it